compliance_checker/cf/cf.py

```diff
@@ -677,6 +685,141 @@ def _split_standard_name(self, standard_name):
         else:
             return standard_name, None
 
+    def appendix_a_validate(self, ds):
```

Comment:
What's the reason for this now being a `check_` method? Won't this make it a little more difficult to adhere to the high-level checker interface?

Reply:
Will move to check instead.